packages/run-protocol/src/vaultFactory/vaultFactory.js

```diff
@@ -43,10 +46,15 @@ const { details: X } = assert;
  *  {electionManager: Instance,
  *   main: {Electorate: ParamRecord<'invitation'>},
  *   timerService: TimerService,
- *   priceAuthority: ERef<PriceAuthority>}>} zcf
+ *   priceAuthority: ERef<PriceAuthority>,
+ *   debtPriceRatio?: Ratio }>} zcf
  * @param {{feeMintAccess: FeeMintAccess, initialPoserInvitation: Invitation}} privateArgs
  */
 export const start = async (zcf, privateArgs) => {
+  const { feeMintAccess, initialPoserInvitation } = privateArgs;
```

Comment:
if it's going to be at the top, might as well destructure in the params,
```suggestion
export const start = async (zcf, { feeMintAccess, initialPoserInvitation }) => {
```